settings/sample-memcache.settings.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Template for memcache configuration and SASL authentication for Acquia Cloud
+ */
+
+if ($is_ah_env) {
+  $settings['cache']['default'] = 'cache.backend.memcache';
+  $settings['memcache']['sasl'] = [
+  'username' => 'user',
```

Comment:
Is there documentation for finding/setting these values? We should probably provide a link.